sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -276,10 +280,18 @@ private static boolean initConfigurations(final @NotNull SentryOptions options)
               });
     }
 
-    final IModulesLoader modulesLoader = options.getModulesLoader();
-    // only override the ModulesLoader if it's not already set by Android
+    final @NotNull IModulesLoader modulesLoader = options.getModulesLoader();
     if (modulesLoader instanceof NoOpModulesLoader) {
-      options.setModulesLoader(new ResourcesModulesLoader(options.getLogger()));
+      if (Platform.isJvm()) {
```

Comment:
`m`: I think we don't need this check, 'cause this was the codepath for JVM platforms anyway, so I think CompositeModulesLoader covers everything already

Reply:
OK Great, I presume this also won't cause problems with other SDKs using the Java SDK, correct?